### PR TITLE
[CodeCompletion] Completion for control trasfer target

### PR DESCRIPTION
**Pull request by rintaro**
Completion after `break` and `continue`

rdar://problem/57016218

### CODE REVIEW

**Comment by rintaro** [comment]
@swift-ci Please smoke test

**Comment by rintaro** [comment]
@swift-ci Please smoke test macOS

**Comment by rintaro** [comment]
@swift-ci Please smoke test macOS

File: lib/IDE/CodeCompletion.cpp [hunk 1]
```diff
@@ -4350,6 +4351,52 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
...
+      SourceLoc TargetLoc;
+      bool IsContinue;
+
+    public:
+      SmallVector<Identifier, 2> Result;
+
+      LabelFinder(SourceManager &SM, SourceLoc TargetLoc, bool IsContinue)
+          : SM(SM), TargetLoc(TargetLoc), IsContinue(IsContinue) {}
+
+      virtual std::pair<bool, Stmt *> walkToStmtPre(Stmt *S) override {
```
Comment (benlangmuir) [inline lib/IDE/CodeCompletion.cpp:4367]: Style nitpick: don't use "virtual" when you have "override".

**Review by benlangmuir — APPROVED** [review]

**Comment by rintaro** [comment]
@swift-ci Please smoke test

**Review by benlangmuir — APPROVED** [review]

**Comment by rintaro** [comment]
@swift-ci Please smoke test macOS

**Pull request merged**